data/p10/pdbg_compatible_propMapping.lsv

```diff
@@ -24,6 +24,9 @@ unit-mem_port:ibm,power10-memport
 lcard-dimm-ddimm:ibm,power10-dimm
```

Comment:
Are these ddimm associations still true?

Reply:
I think we will have either lcard-dimm* or unit_ddr* and both will represent the DIMM.

If yes then, we will call it power10-dimm on the pdbg side. In case the address translation logic needs to be different then we need to differentiate by DDR type.